ChangeLog.d/crl-revocationDate.txt

```diff
@@ -0,0 +1,10 @@
+Security
+   * When checking X.509 CRLs, a certificate was only considered as revoked if
+     its revocationDate was in the past according to the local clock if
+     available. In particular, on builds without MBEDTLS_HAVE_TIME_DATE,
+     certificates were never considered as revoked. On builds with
+     MBEDTLS_HAVE_TIME_DATE, an attacker able to control the local clock (for
+     example, an untrusted OS attacking a secure enclave) could prevent
+     revocation of certificates via CRLs. Fixed by no longer checking the
+     revocationDate field, in accordance with RFC 5280.  Reported and fixed by
+     Raoul Strackx & Jethro Beekman.
```

Comment:
I just realized that the issue was reported independently in https://github.com/ARMmbed/mbedtls/issues/3340 (focusing on builds with `MBEDTLS_HAVE_TIME_DATE`).
```suggestion
     revocationDate field, in accordance with RFC 5280. Reported by
     yuemonangong in #3340. Reported independently and fixed by
     Raoul Strackx and Jethro Beekman in #3433.
```